### ISSUE TITLE
Swipe down to minimize now playing screen

### ISSUE THREAD

**Issue by max5354** [maxrave-dev/SimpMusic#41]
Implement swipe down on thumbnail to minimize now playing screen we need to press button everytime whenever we need to minimum now playing screen.

**Comment by owencz1998** [comment]
Yes definitely need this and swipe the player down to minimise it

**Comment by owencz1998** [comment]
Also @maxrave-dev with the mini media player you could swipe down again on the small player to stop all songs.

**Comment by maxrave-dev** [comment]
swipe right to left miniplayer, you will get close button. Try it. Swipe down to minimize I think hard to implement so may be not planned

**Comment by owencz1998** [comment]
@maxrave-dev that's a shame could u look at innertubes code or would that be to different dude

**Comment by maxrave-dev** [comment]
Hmm, he used Jetpack Compose, by the way, when I created this project, I had zero knowledge about coding and try to learn from basic (eg: XML view). It will be migrated to Jetpack Compose, promised!

**Comment by owencz1998** [comment]
😱 From someone who had no knowledge you have done a incredible job dude.
